pkg/services/accesscontrol/cacheutils.go

```diff
@@ -0,0 +1,26 @@
+package accesscontrol
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/grafana/grafana/pkg/services/auth/identity"
+)
+
+func GetPermissionCacheKey(user identity.Requester) string {
+	return fmt.Sprintf("rbac-permissions-%s", user.GetCacheKey())
+}
```

Comment:
After a quick browse through the code, I don't think this cache ever gets set. Or am I missing something?

Reply:
Hm, seems like it's not set. I'll double check it.

Reply:
Yes, old version used cache set by getUserPermissions, but now it's different, so we cannot rely on it anymore. I added cache set after searching permissions from the DB.